File: automacao/programacao-frc/exemplo-spark.java
import edu.wpi.first.wpilibj.Joystick;
import edu.wpi.first.wpilibj.Spark;

public class Robot extends TimedRobot {
	private final Joystick m_stick = new Joystick(0);
	private final Spark m_spark = new Spark(0);

	@Override
	public void teleopPeriodic() {
		if (m_stick.getRawButton(0)) {
			m_spark.set(0.4);
		} else {
			m_spark.set(0.0);
		}
	}
}

File: automacao/programacao-frc/exemplo-spark.cpp
#include <frc/Joystick.h>
#include <frc/Spark.h>

class Robot : public frc::TimedRobot
{
	frc::Joystick m_stick{0};
	frc::Spark m_motor{0};

public:
	void TeleopPeriodic()
	{
		if (m_stick.GetRawButton(0))
		{
			m_motor.Set(0.4);
		}
		else
		{
			m_motor.Set(0.0);
		}
	}
};

#ifndef RUNNING_FRC_TESTS
int main()
{
	return frc::StartRobot<Robot>();
}
#endif
